styles.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: #7bdede;
+}
```

Comment:
- [OPTIONAL] According to the requirements, the CSS file should include a class, so you should select the element using `class` 👌.
Instead of `h1`, you can use `.heading` for example. (and remember to add this class in your HTML file)
```
.heading {
  color: gold;
}
```

![image](https://user-images.githubusercontent.com/45231276/176363886-c18e5cc2-ce4b-40a2-9002-9ffbd4ec1997.png)